test/e2e/config/docker.yaml

```diff
@@ -237,10 +237,10 @@ variables:
   # allowing the same e2e config file to be re-used in different Prow jobs e.g. each one with a K8s version permutation.
   # The following Kubernetes versions should be the latest versions with already published kindest/node images.
   # This avoids building node images in the default case which improves the test duration significantly.
-  KUBERNETES_VERSION_MANAGEMENT: "v1.27.3"
-  KUBERNETES_VERSION: "v1.27.3"
-  KUBERNETES_VERSION_UPGRADE_FROM: "v1.26.6"
-  KUBERNETES_VERSION_UPGRADE_TO: "v1.27.3"
+  KUBERNETES_VERSION_MANAGEMENT: "v1.28.0"
+  KUBERNETES_VERSION: "v1.28.0"
+  KUBERNETES_VERSION_UPGRADE_FROM: "v1.27.3"
+  KUBERNETES_VERSION_UPGRADE_TO: "v1.28.0"
   ETCD_VERSION_UPGRADE_TO: "3.5.7-0"
```

Comment:
If I saw correctly the new default changed. If yes, let's bump this version as well